packages/shell-api/src/collection.spec.ts

```diff
@@ -265,6 +265,24 @@ describe('Collection', () => {
         );
       });
 
+      it('passes writeConcern through if specified', async() => {
+        serviceProvider.bulkWrite = sinon.spy(() => Promise.resolve({
+          result: { ok: 1 }
+        })) as any;
+
+        await collection.bulkWrite(requests, {
+          writeConcern: { w: 'majority' }
+        });
+
+        expect(serviceProvider.bulkWrite).to.have.been.calledWith(
+          'db1',
+          'coll1',
+          requests,
+          { writeConcern: { w: 'majority' } },
```

Comment:
Do we want to update the code so that we don't send writeConcern twice? We only really need to send it as a command option, not a database option I think

Reply:
But then why do we have code that explicitly adds it to the db options? Like, I’m happy to remove code instead of adding tests if that also works here :)

Reply:
Not sure why that was added but I don't think we still need it, since it seems like all the options in the new driver take in subclasses of `CommandOperationOptions`, which extends `WriteConcernOptions`: https://github.com/mongodb/node-mongodb-native/blob/master/src/operations/command.ts#L18-L20 so it should be enough to just pass in the command options instead of the db options. My guess why it was that way before is that the old driver either didn't document or did not have a way to pass writeConcern via command options

Reply:
Ok, that makes sense to me – maybe we can also use somewhat stricter typescript here to avoid any future confusion, I’ll check and update this accordingly

Reply:
The test is definitely worth keeping btw, just the extra writeConcern in the arguments caught my eye and made me realize maybe we're doing something unnecessary here :)